### DISCUSSION TITLE
multi e-mails notifications

### DISCUSSION THREAD

**Discussion by bzq-pl** [berrystephenw/OctoText#107] [Q&A] [answered]
Hi,
How can I add more than 1 address to send e-mail notifications? Probably I can't do this via Octoprint web Guide but maybe in some config file this is possible?
Regards
Piotr

**Comment by berrystephenw** [comment, marked as answer]
Hi Piotr, Octotext does not allow more than one destination email address. Everything is entered into the setup page - there is no configuration file. It would be possible but would require adding code and testing. For the moment development other than bug fixes are on hold for Octotext. 
 Steve

**Reply by bzq-pl** [reply]
OK. thank you for prompt answer. for now i’ll use forwarding rules in my mailbox (as workaround ;-) )
